test/libsolidity/syntaxTests/emit/emit_struct_storage.sol

```diff
@@ -0,0 +1,10 @@
+contract c {
+    struct S { uint a ; }
+    event E(S);
```

Comment:
What happens if you use `event E(S storage)`?

Reply:
`Expected ',' but got 'storage'`